llvm/lib/Target/NVPTX/NVPTXAsmPrinter.cpp

```diff
@@ -93,20 +95,19 @@ using namespace llvm;
 
 #define DEPOTNAME "__local_depot"
 
-/// DiscoverDependentGlobals - Return a set of GlobalVariables on which \p V
+/// discoverDependentGlobals - Return a set of GlobalVariables on which \p V
```

Comment:
Is this formatting preference documented?

Reply:
Yes https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly